atomic_reactor/util.py

```diff
@@ -1104,7 +1105,12 @@ def get_inspect_for_image(image, registry, insecure=False, dockercfg_path=None):
         else:
             blob_config = json.loads(v1_json['history'][0]['v1Compatibility'])
     else:
-        raise NotImplementedError("No v2 schema 1 image, or v2 schema 2 image or list, found")
+        raise RuntimeError(
+            "Image {image_name} not found: No v2 schema 1 image, "
+            "or v2 schema 2 image or list, found".format(
+                image_name=image
```

Comment:
split this as:
```
        raise RuntimeError("Image {image_name} not found: No v2 schema 1 image, "
                           "or v2 schema 2 image or list, found".format(image_name=image))
```

Reply:
I'm used to use named substitutions because it makes it clean what is expected there and is readable better and it is required for translators (not the case here though).

> Also, consider not passing arguments by name into str.format(), since you're eating up 20+ characters each time you do that and the string formats are not that complicated.

Why is this so bad? It is still better to split long text/code to multiple lines for readability.

Also the indentation you requested makes big empty spaces when functions are long.
```
block:
    block:
         block:
             self.myobject.anotherobject.method("Lorem ipsum dolor sit amet, "
                                                "consectetur adipiscing elit, "
                                                "sed do eiusmod tempor "
                                                "incididunt ut labore et "
                                                "dolore magna aliqua.")
```

This way seems cleaner to me:
```
block:
    block:
         block:
             self.myobject.anotherobject.method(
                "Lorem ipsum dolor sit amet, consectetur adipiscing elit,  sed 
                "do eiusmod tempor incididunt ut labore et dolore magna aliqua."
             )
```

Makes git diffs bigger when change happen.

```diff
- logger.info(
+ logger.warning(
   "Something bad happened"
   "2nd line"
)
```
```diff
- logger.info("Something bad happened"
-             "2nd line")
+ logger.warning("Something bad happened"
+                "2nd line")
```

I will incorporate requested changes given they are PEP8 compatible too.